feat: add properties to DataService data model

bobeal opened the pull request:
* Renamed `endPointURL` to `endpointURL`
* Renamed `endPointDescription` to `endpointDescription`
* Added `disseminationOrganisation`, `dissemination`, `reputationOrganisation`, `reputation` and `accessMechanism` (from Meloda5 metrics)
* Added `imageSource`
* Added `versionNotes`

bobeal committed:
feat: add properties to DataService data model

* Renamed `endPointURL` to `endpointURL`
* Renamed `endPointDescription` to `endpointDescription`
* Added `disseminationOrganisation`, `dissemination`, `reputationOrganisation`, `reputation` and `accessMechanism` (from Meloda5 metrics)
* Added `imageSource`
* Added `versionNotes`

albertoabellagarcia commented on DataService/code/code_for_using_dataModel.DCAT-AP_DataService.py:
this file is generated automatically

albertoabellagarcia commented on DataService/code/code_for_using_pydantic.py:
this file is generated automatically

albertoabellagarcia commented on DataService/examples/example-normalized.json.csv:
this file is generated automatically

albertoabellagarcia commented on DataService/examples/example-normalized.jsonld.csv:
automatically generated file

albertoabellagarcia commented on DataService/examples/example.json.csv:
generated automatically

albertoabellagarcia commented on DataService/examples/example.jsonld.csv:
Generated automatically

albertoabellagarcia commented on DataService/model.yaml:
generated automatically

albertoabellagarcia commented on DataService/schema.sql:
Automatically generated. not need to fix it.

albertoabellagarcia commented on DataService/schemaDTDL.json:
generated automatically

albertoabellagarcia commented on context.jsonld:
generated automatically

albertoabellagarcia reviewed (changes requested):
address the comment, and do not work on automatically generated files.

bobeal commented:
OK removed all the generated files from the PR.

bobeal commented:
@albertoabellagarcia is everything now OK with the PR?

albertoabellagarcia commented:
the data service has an issue the version is published as 1.0 (number) but defined in the schema as string

bobeal committed:
fix: change value of version to string

bobeal commented:
> the data service has an issue the version is published as 1.0 (number) but defined in the schema as string

Fixed. I changed the value of `version` to string.
